spring-cloud-dataflow-server-local/src/main/resources/dataflow-server.yml

```diff
@@ -15,7 +15,7 @@ security:
 
 spring:
   application:
-    name: spring-cloud-dataflow-server
+    name: spring-cloud-dataflow-server-local
```

Comment:
I think this can remain as `spring-cloud-dataflow-server`... if we want it to be consistent regardless of the implementation, for things like config server. WDYT?